crates/fuel-core/src/service/config.rs

```diff
@@ -129,6 +129,7 @@ impl Config {
                 max_queries_depth: 16,
                 max_queries_complexity: 20000,
                 max_queries_recursive_depth: 16,
+                max_queries_body_limit: 16 * 1024 * 1024,
```

Comment:
Currently we support queries up to 7.5mb at the load balancer level. Should we match that here?

I think it's pretty common for RPC providers to support up to 10+ mb responses on other jsonrpc based chains.

cc @mchristopher